filebeat/harvester/config.go

```diff
@@ -20,8 +21,10 @@ var (
 		BackoffFactor:   2,
 		MaxBackoff:      10 * time.Second,
 		CloseOlder:      1 * time.Hour,
-		ForceCloseFiles: false,
 		MaxBytes:        10 * (1 << 20), // 10MB
+		CloseRemoved:    false,
+		CloseRenamed:    false,
+		ForceCloseFiles: false,
```

Comment:
@urso I want to make the migration to 5.0 as seamless as possible. That is why I would prefer to keep this in.